### PR TITLE
zh: add description of task config `ignore-checking-items`

### PR DESCRIPTION
**Pull request by lance6716**
### What is changed, added or deleted? (Required)

add description of task config `ignore-checking-items`, part of https://github.com/pingcap/dm/issues/200

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now) (**Please help cherry-pick**)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/dm/issues/200

### CODE REVIEW

File: zh/config-overview.md [hunk 1]
```diff
@@ -35,3 +35,19 @@ DM 配置的关键概念如下：
 | :------------ | :------------ | :------------------ |
 | source-id  | 唯一确定一个 MySQL 或 MariaDB 实例，或者一个具有主从结构的复制组，字符串长度不大于 32 | `inventory.ini` 的 `source_id`；<br/> `dm-master.toml` 的 `source-id`；<br/> `task.yaml` 的 `source-id` |
 | DM-worker ID | 唯一确定一个 DM-worker（取值于 `dm-worker.toml` 的 `worker-addr` 参数） | `dm-worker.toml` 的 `worker-addr`；<br/> dmctl 命令行的 `-worker` / `-w` flag |
+
+### 关闭检查项
```
Comment (GMHDBJD) [inline zh/config-overview.md:39]: How about move this to (precheck.md)[https://github.com/pingcap/docs-dm/blob/release-1.0/zh/precheck.md]

Comment (lance6716) [reply zh/config-overview.md]: done

**Comment by csuzhangxc** [comment]
@toutdesuite PTAL

File: zh/task-configuration-file-full.md [hunk 2]
```diff
@@ -13,6 +13,10 @@ aliases: ['/docs-cn/dev/reference/tools/data-migration/configure/task-configurat
 
 关于包括 `source-id` 和 DM-worker ID 在内的关键概念的介绍，请参阅[关键概念](config-overview.md#关键概念)。
 
+## 关闭检查项
+
+DM 会根据任务类型进行相应检查，用户可以在任务配置文件中使用`ignore-checking-items`配置关闭检查，请参阅[关闭检查项](precheck.md#关闭检查项)
```
Comment (toutdesuite) [inline zh/task-configuration-file-full.md:18]:
```suggestion
DM 会根据任务类型进行相应检查。可以参考[关闭检查项](precheck.md#关闭检查项)，在任务配置文件中使用 `ignore-checking-items` 配置关闭相应检查。
```

File: zh/precheck.md [hunk 3]
```diff
@@ -61,3 +61,19 @@ aliases: ['/docs-cn/dev/reference/tools/data-migration/precheck/','/docs-cn/v3.1
             - 分表存在自增主键，自增主键 column 类型为 bigint，但没有为其配置列值转换
 
         - 其他情况下检查将成功
+
+### 关闭检查项
+
+DM 会根据任务类型进行相应检查，用户可以在任务配置文件中使用`ignore-checking-items`配置关闭检查。`ignore-checking-items`是一个列表，其中可能的取值包括：
```
Comment (toutdesuite) [inline zh/precheck.md:67]:
```suggestion
DM 会根据任务类型进行相应检查，用户可以在任务配置文件中使用 `ignore-checking-items` 配置关闭检查。`ignore-checking-items` 是一个列表，其中可能的取值包括：

```

File: zh/precheck.md [hunk 4]
```diff
@@ -61,3 +61,19 @@ aliases: ['/docs-cn/dev/reference/tools/data-migration/precheck/','/docs-cn/v3.1
...
+| 取值   | 含义   |
+| :----  | :-----|
+| all | 关闭所有检查 |
+| dump_privilege | 关闭检查上游 MySQL 实例用户的 dump 相关权限 |
+| replication_privilege | 关闭检查上游 MySQL 实例用户的 replication 相关权限 |
+| version | 关闭检查上游数据库版本 |
+| binlog_enable | 关闭检查上游数据库是否已启用 binlog |
+| binlog_format | 关闭检查上游数据库 binlog 格式是否为 row |
```
Comment (toutdesuite) [inline zh/precheck.md:75]:
```suggestion
| binlog_format | 关闭检查上游数据库 binlog 格式是否为 ROW |
```

**Pull request merged**